processing/data_collection/gazette/pipelines.py

```diff
@@ -1,17 +1,19 @@
+import hashlib
+import magic
 import os
+import pathlib
 import subprocess
-import hashlib
 
-import magic
 from database.models import Gazette, initialize_database
+from gazette.settings import FILES_STORE
+from itemadapter import ItemAdapter
 from scrapy.exceptions import DropItem
+from scrapy.http import Request
+from scrapy.pipelines.files import FilesPipeline
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.orm import sessionmaker
 
 
-from gazette.settings import FILES_STORE
-
-
```

Comment:
A [PEP8 sugere que a gente separe os `import`s em 3 blocos](https://www.python.org/dev/peps/pep-0008/#imports): bibliotecas nativas, bibliotecas de terceiros e módulos da própria aplicação.

> Imports should be grouped in the following order:
>
> * Standard library imports.
> * Related third party imports.
> * Local application/library specific imports.
>
> You should put a blank line between each group of imports.

Sugiro essas mudanças para passar o `import magic` (que acho que é de terceiros) para o segundo bloco, e o `gazette` e o `database` para o terceiro bloco:


```python
import hashlib
import os
import pathlib
import subprocess

import magic
from itemadapter import ItemAdapter
from scrapy.exceptions import DropItem
from scrapy.http import Request
from scrapy.pipelines.files import FilesPipeline
from sqlalchemy.exc import IntegrityError
from sqlalchemy.orm import sessionmaker

from database.models import Gazette, initialize_database
from gazette.settings import FILES_STORE
```

Reply:
[There is a library](https://pypi.org/project/isort/) that does that automatically @jvanz . You would just need to apply `black` afterwards.